<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatosMaestrosDialog</class>
 <widget class="QDialog" name="DatosMaestrosDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datos Maestros de Productos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tabla_productos">
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <column>
      <property name="text">
       <string>Tipo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Unidad de Medida</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Precio Unitario</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnIngresar">
     <property name="text">
      <string>Ingresar Nuevo Producto</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bt_guardar">
     <property name="text">
      <string>Almacenar Productos</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>